instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,57 @@
+
+import React from "react"
+import Comment from "./Comment.js"
+import PropTypes from "prop-types";
+import CommentForm from "./CommentForm"
+
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments
+    }
+  }
+
+  addNewComment = (text) => {
+    const newComment = {
+      id: Date.now(),
+      username: "placeholder",
```

Comment:
Make this more robust! Set the username to the username we put in when we log in.
username: localStorage.getItem("username"),